Cutelyst/Plugins/Memcached/memcached.h

```diff
@@ -58,7 +58,7 @@ class MemcachedPrivate;
  * @li @a compression_level - integer value, the compression level used by qCompress (default: -1)
  * @li @a compression_threshold - integer value, the compression size threshold in bytes, only input values bigger than the threshold will be compressed (default: 100)
  * @li @a encryption_key - string value, if set and not empty, AES encryption will be enabled (default: empty)
- * @li @a sasl_user - string value, if set and not empty, SASL authentication will be used (default: empty)
+ * @li @a sasl_user - string value, if set and not empty, SASL authentication will be used - note that SASL support has been enabled when building libmemcached (default: empty)
```

Comment:
Shouldn't it be:
note that SASL support has to be enabled when building....?

Reply:
ouh, yes 🙈